src/components/calculator.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+
+class Calculator extends Component {
+	
+	constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container-fluid">
+        <div className="row">
+          <div className="col bg-secondary text-end border-secondary text-white">0</div>
+        </div>
+				<div className="row">
+          <div className="col-3 bg-light text-center border-secondary border border-1">AC</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">+/-</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">%</div>
+					<div className="col-3 bg-warning text-center border-secondary border border-1">/</div>
+        </div>
+				<div className="row">
+          <div className="col-3 bg-light text-center border-secondary border border-1">7</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">8</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">9</div>
+					<div className="col-3 bg-warning text-center border-secondary border border-1">*</div>
+        </div>
+				<div className="row">
+          <div className="col-3 bg-light text-center border-secondary border border-1">4</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">5</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">6</div>
+					<div className="col-3 bg-warning text-center border-secondary border border-1">-</div>
+        </div>
+				<div className="row">
+          <div className="col-3 bg-light text-center border-secondary border border-1">1</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">2</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">3</div>
+					<div className="col-3 bg-warning text-center border-secondary border border-1">+</div>
+        </div>
+				<div className="row">
+          <div className="col-6 bg-light text-center border-secondary border border-1">0</div>
+					<div className="col-3 bg-light text-center border-secondary border border-1">.</div>
+					<div className="col-3 bg-warning text-center border-secondary border border-1">=</div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
- [ ] Please make sure to adjust the styling and spacing of your design to better match the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/211679198-76076755-3203-44f5-ad56-8fc0c58c0c63.png) | ![image](https://user-images.githubusercontent.com/62290545/211679254-8ab8c243-3b7f-43d3-a14f-c28c60489574.png) |